api/comments_api.py

```diff
@@ -61,6 +61,7 @@ def do_get(self, **kwargs) -> dict[str, list[dict[str, Any]]]:
     """Return a list of all review comments on the given feature."""
     feature_id = kwargs['feature_id']
     gate_id = kwargs.get('gate_id', None)
+    # TODO(jrobbins): Remove comments_only after next deployment.
     comments_only = self.get_bool_arg('comments_only')
```

Comment:
```suggestion
    comments_only = self.get_bool_arg('comments_only')
```
comments_only needs to be removed here, since it is removed on the client side. Same goes for`Activity.get_activities(
        feature_id, gate_id, comments_only=comments_only)`

Reply:
OK, I removed this.   I had thought that it was needed by anyone who had the old SPA open in a browser tab because the old SPA would still send comments_only, but it is OK for the server to simply ignore it.